### ISSUE TITLE
Solve a Clustering problem ✨ !!

### ISSUE THREAD

**Issue by BlueBlaze6335** [gdscjgec/ML-Gallery#4]
### Add a new Folder - "Clustering" and add a notebook for the solution to any clustering dataset !!

**Choose and download your dataset from [here](https://github.com/gdscjgec/ML-Gallery/blob/main/Datasets.md) or you can add any.**

After downloading the dataset , you can work on that in an ide or a notebook. Solve the problem with proper EDA and data engineering. Use any clustering algorithm you think is cool and submit a README.md file with the brief documentation of your project along with the .py/.ipynb file.

***If you are using any other Dataset , don't for get to add it in [this](https://github.com/gdscjgec/ML-Gallery/blob/main/Datasets.md).***

**Comment by Yashesvinee** [comment]
Hi can I work on this?

**Comment by BlueBlaze6335** [comment]
@Yashesvinee yes you can start working on this issue !! Please don't forget to add README with proper documentation !! All the best !!

**Comment by varun290101** [comment]
can you assign this to me?

**Comment by BlueBlaze6335** [comment]
@varun290101 i am assigning this to you !! Happy contributing !!

**Comment by khusheekapoor** [comment]
Hey, please assign this to me in R. Thanks!

**Comment by BlueBlaze6335** [comment]
@khusheekapoor okay !! please follow the rules and create a separate folder for R !!

**Comment by BlueBlaze6335** [comment]
@Kasrazeba-Mulla 🙌🏻

**Comment by BlueBlaze6335** [comment]
Everyone please check the discussion section !!

**Comment by AshiJoshi** [comment]
@BlueBlaze6335 I want to work on this issue to cluster the dataset of tree species

**Comment by BlueBlaze6335** [comment]
@AshiJoshi please go through the repo and do as others have done !! I am assigning you !!
